examples/ASDKgram/Sample/AppDelegate.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AppDelegate.h
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 2/16/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@protocol PhotoFeedViewControllerProtocol <NSObject>
```

Comment:
I should have named it more clearly. This is used by both the ViewController and NodeController to reset their data sources when switching between tabs.